myapp/app/controllers/tasks_controller.rb

```diff
@@ -2,19 +2,31 @@
 
 class TasksController < ApplicationController
   before_action :set_task, only: [:show, :edit, :update, :destroy]
+  TASKS_PER_PAGE = 5
 
   def index
-    if params[:search]
-      @tasks = Task.search_title(params[:search])
-    elsif params[:status] && Task.statuses.include?(params[:status])
+    # For search function
+    if params[:search] && !params[:status].blank?
```

Comment:
any better way to write this if (L8-L15) block?

Reply:
How about this?
```
    if params[:search]
      if !params[:status].blank?
        @tasks = Task.filter_status(params[:status]).search_title(params[:search])
      else
        @tasks = Task.search_title(params[:search])
      end
    else
      @tasks = Task.all
    end
```